docs/_documentations/workingwithtemplates.md

```diff
@@ -91,6 +91,8 @@ Add your template sources to Codewind with the **Template Source Manager**.
 1. Add the template.
    - **VS Code:** In the **Template Source Manager**, click **Add New**.
    - **Eclipse:** Right-click the connection in the Codewind Explorer view and select **Manage Template Sources...**. After the **Manage Template Sources** wizard appears, click **Add...**.
+   - **IntelliJ:** Right-click the connection in the Codewind Explorer view and select **Manage Template Sources...**. After the **Manage Template Sources** wizard appears, select **My Private Template Repository** and click **OK**.
```

Comment:
Keep this the same as Eclipse.  My screenshot just showed the example Repo that was created.